sidebars.js

```diff
@@ -103,6 +103,7 @@ module.exports = {
             'guides/connecting_to_your_data/how_to_choose_which_dataconnector_to_use',
             'guides/connecting_to_your_data/how_to_choose_between_working_with_a_single_or_multiple_batches_of_data',
             'guides/connecting_to_your_data/how_to_configure_an_inferredassetdataconnector',
+            'guides/connecting_to_your_data/datasource_configuration/how_to_configure_a_pandas_datasource',
```

Comment:
I actually had some trouble finding the document in the netlify-rendered website.. Is this a new section under "connecting to your data?"

Reply:
The folder `datasource_configuration` is new, but currently its contents are being put in the core skills section under connecting to your data.  That's probably going to see some overhaul once all the docs are done, as some of the older ones will no longer be needed since they will be duplicated by the content in these new ones.